File: python/resistor-color-duo/resistor_color_duo.py
color_list = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]


def value(colors):
    code = ''
    for color in colors:
        if color in color_list:
            code += str(color_list.index(color))
    return int(code[:2])

File: typescript/resistor-color-duo/resistor-color-duo.ts
enum Colors {
  black,
  brown,
  red,
  orange,
  yellow,
  green,
  blue,
  violet,
  grey,
  white,
}

export const decodedValue = (color: (keyof typeof Colors)[]): number =>
  parseInt(`${Colors[color[0]]}${Colors[color[1]]}`);
